code/modules/research/stock_parts.dm

```diff
@@ -273,6 +283,48 @@ If you create T5+ please take a pass at gene_modder.dm [L40]. Max_values MUST fi
 	rating = 4
 	materials = list(/datum/material/iron=80)
 
+//Rating 5
+
+/obj/item/stock_parts/capacitor/cubic
+	name = "cubic capacitor"
+	desc = "A cubic capacitor used in the construction of a variety of devices."
+	icon_state = "cubic_capacitor"
+	rating = 5
+	materials = list(/datum/material/iron=50, /datum/material/glass=50)
+	resistance_flags = INDESTRUCTIBLE
```

Comment:
why

Reply:
to prevent atmospherics from killing parts on creation